app/controllers/assessments_controller.rb

```diff
@@ -1982,6 +1982,7 @@ def get_props
   def get_assessment
     @assessment = Assessment.find params[:assessment_id] || params[:id]
     @course = @assessment.course
+    @breadcrumbs << (view_context.current_assessment_link)
```

Comment:
view_context lets you use view helper functions (like link_to, form_for, etc.) inside of controller code.  This is calling the 'current_assessment_link' helper method.